tokio-io/src/codec/mod.rs

```diff
@@ -10,6 +10,9 @@
 //! [`Stream`]: #
 //! [transports]: #
 
+#![doc(hidden)]
```

Comment:
While `length_delimited` hasn't been moved to tokio-util in this PR, I think it is still fine to just deprecate it here.

Reply:
Sounds good. I removed that part.